Zend/zend_portability.h

```diff
@@ -545,12 +545,12 @@ static zend_always_inline double _zend_get_nan(void) /* {{{ */
  * __attribute__((target("avx2"))) functions until gcc 4.9.
  */
 # if PHP_HAVE_AVX2_INSTRUCTIONS && defined(HAVE_IMMINTRIN_H) && \
-  (defined(__llvm__) || defined(__clang__) || (defined(__GNUC__) && ZEND_GCC_VERSION >= 4009))
+  (defined(__llvm__) || defined(__clang__) || (defined(__GNUC__) && ZEND_GCC_VERSION >= 4009)) && !defined(__OpenBSD__)
 # define PHP_HAVE_AVX2
 # endif
 #endif
 
-#ifdef __SSSE3__
+#if defined(__SSSE3__) && !defined(__OpenBSD__)
```

Comment:
Is this one needed? These checks are relevant for `-march=native` builds, where we just directly compile for the SIMD target and don't do any of the multi-versioning and runtime dispatch. Is that broken as well?

Reply:
I m afraid so.